presto-hive/src/main/java/com/facebook/presto/hive/orc/OrcPageSourceFactory.java

```diff
@@ -206,6 +207,9 @@ public static OrcPageSource createOrcPageSource(
             if (e.getClass().getSimpleName().equals("BlockMissingException")) {
                 throw new PrestoException(HIVE_MISSING_DATA, message, e);
             }
+            else if (e.getClass().getSimpleName().equals("InvalidProtocolBufferException")) {
```

Comment:
Then we need to add `com.facebook.presto.orc:orc-protobuf:jar` to `presto-hive` dependencies, do we want to do that for this single line change?